### PR TITLE
Mobile menu not scroll on older devices

### PR DESCRIPTION
**Pull request by umbobabo**
Refering to this issue https://github.com/economist-components/component-navigation/issues/36

### CODE REVIEW

**Comment by fabiosantoscode** [comment]
:rainbow: from me. @gcedo take the second look

File: test/index.es6 [hunk 1]
```diff
@@ -96,14 +96,16 @@ describe(`A navigation`, () => {
...
         sectionsCardData: context,
         accordionData: accordionContext,
         sharedMenu,
       });
-      instance.renderLoginLogout()
-        .props.href.should.equal('/logout?destination=%2Ffoo%2Fbar');
+      const listElements = instance.renderLoginLogout().props.children.props.children;
+      const loginLink = listElements.slice(-1)[0];
+      const href = loginLink.props.children.props.href;
+      href.should.equal(`/logout?destination=${decodeURIComponent('%2Ffoo%2Fbar')}`);
```
Comment (fabiosantoscode) [inline test/index.es6]: Actually no, this shouldn't be exactly `/foo/bar` but instead should be encoded.

File: test/index.es6 [hunk 2]
```diff
@@ -96,14 +96,16 @@ describe(`A navigation`, () => {
...
         sectionsCardData: context,
         accordionData: accordionContext,
         sharedMenu,
       });
-      instance.renderLoginLogout()
-        .props.href.should.equal('/logout?destination=%2Ffoo%2Fbar');
+      const listElements = instance.renderLoginLogout().props.children.props.children;
+      const loginLink = listElements.slice(-1)[0];
+      const href = loginLink.props.children.props.href;
+      href.should.equal(`/logout?destination=/foo/bar`);
```
Comment (fabiosantoscode) [inline test/index.es6]: Nope, we want %2Ffoo%2Fbar. It should be urlencoded.

**Comment by umbobabo** [comment]
@keithamus If you moved this implementation on another PR can we close this one?

**Comment by keithamus** [comment]
@umbobabo this bit: `max-height: 400px;` is not in my PR, but the test fixes are. If you want `max-height: 400px;` to get in you could rebase your PR and remove the test fixes, if not then its fine to close :smile:

**Comment by fabiosantoscode** [comment]
Just to be clear here, the tests aren't broken. The tests are correct, and are failing as they should. The URL inserted into the "destination" querystring parameter is indeed not URL encoded at the time.

**Comment by keithamus** [comment]
@fabiosantoscode More accurately; existing tests are broken and this is the wrong fix. See https://github.com/economist-components/component-navigation/pull/38/files#diff-910eb6f57886ca16c136101fb1699231R105 which has the right test fix.

**Comment by keithamus** [comment]
I'm going to close this - @umbobabo if you want the max-height fix in, please raise a new PR. Thank you very much :smile:

**Pull request closed without merging**